packages/editor/src/lib/editor/managers/WatermarkManager.ts

```diff
@@ -1,11 +1,19 @@
+import w from '../../../watermark.png'
 import { TL_CONTAINER_CLASS } from '../../TldrawEditor'
+import { getDefaultCdnBaseUrl } from '../../utils/assets'
 import { Editor } from '../Editor'
 import { LicenseFromKeyResult } from './LicenseManager'
+const WATERMARK_FILENAME = 'watermark.png'
 
 export class WatermarkManager {
 	constructor(private editor: Editor) {}
 	private createWatermark() {
-		const watermark = document.createElement('a')
+		const watermark = document.createElement('img')
+		if (navigator.onLine) {
```

Comment:
How reliable is this? We could also react to changes to this status, but not sure if it's needed?

Reply:
i think it's good enough for now!